### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Visual Requirement] [MAS1.4.3][Luminosity ratio is less than 4.5:1 for the several controls of the page.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#711]
**User Experience:**
User with partial vision disability will not be able to view the controls properly as the luminosity ratio is less than 4.5:1

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tool :Color Contrast Analyser

**Attachments:**
![mas1 4 3 description](https://user-images.githubusercontent.com/33054846/32598342-96b29b08-c55f-11e7-9392-53781ba9cb97.png)
![mas1 4 3 command bar title](https://user-images.githubusercontent.com/33054846/32598343-97a7e3f6-c55f-11e7-986a-0f244491fe80.png)
![mas1 4 3 warning message link](https://user-images.githubusercontent.com/33054846/32598346-98f2424c-c55f-11e7-8140-d338c59058b7.png)
![mas1 4 3 line number](https://user-images.githubusercontent.com/33054846/32598350-9b7569cc-c55f-11e7-8e2b-51198e72d17a.png)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Navigate to Command bar and verify the luminosity ratio.
3. Navigate to description text of the gallery list and verify the luminosity ratio.
4. Navigate to the warning message link and verify the luminosity ratio.
5. Navigate to any editor and verify the luminosity ratio for the Line numbers.

**Current Behavior:**
Luminosity ratio for the Command bar is 4.27:1.
Luminosity ratio for the description text is 3.69:1
Luminosity ratio for the warning message link is 4.09:1
Luminosity for the Editor Line numbers is 2.42:1

**Expected Behavior:**
Luminosity ratio should be more than or equal to 4.5:1 for all the controls.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.4.3
